### DISCUSSION TITLE
Test and play against Custom Engine

### DISCUSSION THREAD

**Discussion by vitorfrois** [lichess-bot-devs/lichess-bot#860] [Ideas]
Is it possible that one can play against Homemade Custom Engine with python as described in [Create Chess Engine](https://github.com/lichess-bot-devs/lichess-bot/wiki/Create-a-custom-engine).
I don't know if this feature should be implemented in this repository, but at least it would be interesting to have a guide explaining how to do it.

**Comment by MarkZH** [comment]
When you say "play against," are you asking if you could play against your homemade bot? If so, you can login to lichess with your own account and challenge your bot after starting lichess-bot.
